pkg/objstore/s3/s3.go

```diff
@@ -202,3 +205,57 @@ func (b *Bucket) Delete(ctx context.Context, name string) error {
 	b.opsTotal.WithLabelValues(opObjectDelete).Inc()
 	return b.client.RemoveObject(b.bucket, name)
 }
+
+// IsObjNotFoundErr returns true if error means that object is not found. Relevant to Get operations.
+func (b *Bucket) IsObjNotFoundErr(err error) bool {
+	return minio.ToErrorResponse(err).Code == "NoSuchKey"
+}
+
+func configFromEnv() *Config {
+	c := &Config{
+		Bucket:    os.Getenv("S3_BUCKET"),
```

Comment:
Is there anyway we can get around using environment vars in tests?

Reply:
not really ): We need custom parameters to have tests against real s3 right?